doc/src/Howto_github.rst

```diff
@@ -480,11 +480,11 @@ Some recent changes to the workflow are not captured in this tutorial.
 For example, in addition to the *develop* branch, to which all new
```

Comment:
Once again I think that having all of this information scattered in different docs is not ideal...

Reply:
Neither is having everything combined where fewer people will see it.